python-sdk/tests/sql/operators/test_cleanup.py

```diff
@@ -42,16 +39,19 @@
     {
         "database": Database.SNOWFLAKE,
     },
+    {
+        "database": Database.REDSHIFT,
+    },
 ]
 SUPPORTED_DATABASES_WITH_FILE = [dict(x, **{"file": File(DEFAULT_FILEPATH)}) for x in SUPPORTED_DATABASES]
 
 
 @pytest.mark.integration
 @pytest.mark.parametrize(
     "database_table_fixture",
-    SQLITE_ONLY,
+    SUPPORTED_DATABASES_WITH_FILE,
     indirect=True,
-    ids=["sqlite"],
+    ids=["sqlite", "postgres", "bigquery", "snowflake", "redshift"],
```

Comment:
Any reason why we're not using `astro_sdk.constants.SUPPORTED_DATABASES` here, and in other similar test cases?

Reply:
Because I didn't know that we have this :) Thanks :)